### ISSUE TITLE
Quick fix support?

### ISSUE THREAD

**Issue by activey** [Prince781/vala-vscode#10]
Hi,

Just wondering, any chances for getting quick-fix support? :)
Like for example for creating nonexisting methods, type constructors with non existing parameters, etc... , eg:

When I have a type SomeType with default non-parameter constructor and I want to call it like this:
```java
new SomeType("string value")
```
It would be great to make vscode automatically fix it by creating constructor with string parameter type in SomeType :)

**Comment by Prince781** [comment]
Yes, definitely. This is something planned, but it's a feature to implement in the language server itself with the [`textDocument/codeAction` API](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_codeAction).
